Corona-Warn-App/build.gradle

```diff
@@ -318,7 +318,7 @@ dependencies {
     implementation 'androidx.navigation:navigation-ui-ktx:2.2.2'
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation 'androidx.preference:preference:1.1.1'
-    implementation 'androidx.work:work-runtime-ktx:2.5.0-beta01'
+    implementation 'androidx.work:work-runtime-ktx:2.4.0'
```

Comment:
Are both issues related or why are both changes in the same PR?

Reply:
tickets were declared to related inside jira.
I can separte them if needed.